SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/Service/MetadataSupervisor.cs

```diff
@@ -222,10 +222,11 @@ private void TranslationService_TranslationReceived(TranslationData translationD
 			}
 			else if (ActiveDocument is not null)
 			{
-				foreach (var sourceSegment in translationData.SourceSegments)
+				foreach (var sourceSegment in translationData.Segments)
```

Comment:
Is it possible for Segments to be null?

Reply:
No, not here... we reach this method because a translation was received and it will always have those data sets - they are coming from SDLMTCloudLanguageDirection class, from the TUs sent by Trados for translation